README.md

```diff
@@ -1 +1,11 @@
 # hw01
+# Homework assignment
+## Emma Gilheany
+### Week 1 Comp for Social Science
+
+I am a 5th year PhD student in Anthropology who has **absolutely no** programming experience. My dissertation is a *qualitative and creative* humanistic inquiry into the recent history of Hopedale, Nunatsiavut. Here is a [somewhat outdated summary](https://cissr.uchicago.edu/research/2018-2019/field-research/emma-gilheany) of what I am working on. Lately, I am fascinated by
```

Comment:
Good use of list, and very interesting profile! I didn't know those topics were covered by the study of Anthropology.